runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListTable/UsersListTable.tsx

```diff
@@ -44,6 +46,7 @@ const UsersListTable: FC<UsersListTableProps> = ({
                         pageSize={pageSize}
                         onPageSizeChange={(newPageSize) => onPageSizeChange(newPageSize)}
                         disableSelectionOnClick
+                        onRowClick={({ row }) => openUserEditDialog(row)}
```

Comment:
imo by clicking at the record it's expected to see detailed info, not being able to edit user data directly. Wrench icon/or this:
<img width="226" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/d97db8c1-11c7-4d05-8815-78bdc2f5ed15">
is more intuitive for me